packages/backend/src/open_payments/route-errors.ts

```diff
@@ -0,0 +1,79 @@
+import { Logger } from 'pino'
+import { AppContext } from '../app'
+import { OpenAPIValidatorMiddlewareError } from '@interledger/openapi'
+import { WalletAddressSubresource } from './wallet_address/model'
+
+export class OpenPaymentsServerRouteError extends Error {
```

Comment:
@njlie what do you think about having something like this in the `auth` package where we would define a custom `GNAPError` class and catch it similarily in a middleware?